src/NuGet.Core/NuGet.Packaging/Signing/Content/KeyPairFileReader.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace NuGet.Packaging.Signing
 {
-    public class KeyPairFileReader : IDisposable
+    public sealed class KeyPairFileReader : IDisposable
```

Comment:
In some places you add an overload, in others you mark the class as sealed, any particular reasoning?

Same feedback as earlier applies, marking something as sealed can be a breaking change.

Reply:
> In some places you add an overload, in others you mark the class as sealed, any reasoning?

I added overload method, if the class that implements `IDisposable` has been inherited in the product code otherwise marked it as sealed. [reference](https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose#cascade-dispose-calls)

It didn't strike me that this could be a breaking change. I will add overload method instead of marking class as `sealed`.

Reply:
I have removed `sealed` keyword from classes in core libraries and added a protected member instead to dispose objects.